routers/chats/utilities/session.py

```
@@ -0,0 +1,361 @@
+
+            sentiments = {'Neutral': 0, 'Positive': 0, 'Negative': 0}
+            for score in record['roles']:
+                if score['type'] == 'human':
+                    try:
+                        if score['sentiment'] == 'Neutral':
+                            sentiments['Neutral'] += 1
+                        elif score['sentiment'] == 'Positive':
+                            sentiments['Positive'] += 1
+                        elif score['sentiment'] == 'Negative':
+                            sentiments['Negative'] += 1
+                    except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)